realm/realm-library/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -21,20 +21,32 @@
 /**
  * The LinkView class represents a core {@link RealmFieldType#LIST}.
  */
-public class LinkView extends NativeObject {
+public class LinkView implements NativeObject {
 
     private final Context context;
     final Table parent;
     final long columnIndexInParent;
+    private final long nativePointer;
```

Comment:
since the interface is `getNativePtr`, so use `nativePtr` for all looks better. will change in that way.